pyproject.toml

```diff
@@ -3,19 +3,34 @@ name = "datadog_lambda"
 version = "8.121.0.dev0"
 description = "The Datadog AWS Lambda Library"
 authors = ["Datadog, Inc. <dev@datadoghq.com>"]
+
+[project]
+name = "datadog_lambda"
+version = "8.121.0.dev0"
```

Comment:
Looks like we now have the version declared in two places. Do we need both? I'm wondering if we can potentially remove the `tools.poetry` section above.